Source/Extensions/Blazorise.DataGrid/_DataGridCellEditValidation.razor

```diff
@@ -1,8 +1,19 @@
 ﻿@typeparam TItem
 @inherits _DataGridCellEdit<TItem>
```

Comment:
Should this inherit `_DataGridCellEdit<TItem>` or `_DataGridCellEditValidation<TItem>`?
I don't think it quite matters though.

Reply:
No this is good. _DataGridCellEditValidation is partial class the inherits from _DataGridCellEdit

Reply:
Oh I did not notice you switched to using partial class. You used to do inherit as a way of merging the razor + Cs file.

Reply:
Inherit was used before Blazor supported partials. But I changed it couple of months ago.